src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -145,13 +143,14 @@ public void Start(ApplicationData applicationData)
                 Y = Pos.Top(filterLabel),
                 Clicked = () =>
                 {
-                    FilterData(filterField.Text.ToString());
-                    _listView.SetSource(_itemList);
-                    RefreshSelectedListView();
+                    FilterField_Changed(null, filterField.Text);
                 }
             };
 
-            var header = new Label(GetPaddedString(gridHeaders, _listViewOffset + _listViewOffset - 1))
+            var header = new Label(GridViewHelpers.GetPaddedString(
```

Comment:
Can we abstract the call to `GetPaddedString()` as gridHeaders.ToString()?

Reply:
It would still take 2 parameters because of `_listViewOffset + _listViewOffset - 1` and  `_listViewColumnWidths`... is that really worth having an extensions class for `List<string>`?

Reply:
We can defer this.  We may eventually need the headers to be buttons to support click sorting.

Reply:
I added some bullet points to the description of things I've been thinking of.